lib/utils/KalturaUtils.js

```diff
@@ -301,6 +301,10 @@ KalturaUtils = {
 	},
 
 	downloadHttpUrl : function(urlStr, options, successCallback, errorCallback) {
+		if (!urlStr || (typeof urlStr !== 'string' && urlStr instanceof String)){
```

Comment:
in node Js both are treated as Strings yet are different arguments since we want the argument to be string we need to verify it